integration-tests/tests/auth_test.go

```diff
@@ -59,11 +59,11 @@ func (s *authSuite) SetUpTest(c *check.C) {
 func (s *authSuite) TestRegressionAuthCrash(c *check.C) {
 	cmd := []string{"snap", "install", "hello-world"}
 	output, _ := cli.ExecCommandErr(cmd...)
-	c.Assert(output, testutil.Contains, `error: access denied`)
+	c.Assert(output, testutil.Contains, `Access denied (snap login --help).`)
```

Comment:
Comparing the diff, I think this should actually be "error: access denied (snap login --help)", to conform to our other error messages.